packages/i18nify-js/README.md

```diff
@@ -8,7 +8,7 @@ And hey, hang tight—I’ll break down each of these enchanting modules in the
 
 ## Install
 
-`yarn add i18nify`
+`yarn add @razorpay/i18nify-js`
```

Comment:
question: can't we install this directly `yarn add 18nify-js` like this?

Reply:
No, the package is named as `@razorpay/i18nify-js`. This must have been missed in the documentation before.